### PR TITLE
EMP-275-access-cookies

### PR DESCRIPTION
**Pull request by BNmoj**
Changes - 

- created cookies.njk
- created staticPageController.ts
- added staticPageController.ts to controllers/index.ts 
- added staticPageController to routes/index.ts
- added cookies link to footer.njk

### CODE REVIEW

File: server/routes/index.ts [hunk 1]
```diff
@@ -82,5 +83,7 @@ export default function routes({
 
   get('/download-evidence', downloadEvidenceController.download())
 
+  get('/cookies', staticPageController.showCookies())
```
Comment (muyinatech) [inline server/routes/index.ts:86]: unit test is missing

File: server/controllers/staticPageController.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { type Request, type RequestHandler, type Response } from 'express'
```
Comment (muyinatech) [inline server/controllers/staticPageController.ts:1]: unit test is missing

File: server/routes/index.ts (hunk 1 above)
Comment (BNmoj) [reply server/routes/index.ts:86]: added

File: server/controllers/staticPageController.ts (hunk 2 above)
Comment (BNmoj) [reply server/controllers/staticPageController.ts:1]: added

**Review by muyinatech — APPROVED** [review]

**Pull request merged**